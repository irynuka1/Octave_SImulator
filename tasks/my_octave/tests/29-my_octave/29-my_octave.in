L 2 3
5 1 5
5 4 4
L 3 5
1 1 2 3 2
2 1 3 3 2
2 5 3 4 3
L 3 2
2 4
1 3
2 2
L 4 5
3 3 5 1 5
4 2 3 4 4
5 2 2 4 4
3 5 3 3 3
M 0 2
R 4 2
P 4
R -1 5
O
F 3
P 3
M 0 1
P 4
R 4 -1
P 4
M 4 4
P 5
R 4 2
P 4
T 4
F 2
D 4
R 0 3
C 0
2
0 1
2
0 1
P 0
R 0 4
P 0
F 0
T 3
Q