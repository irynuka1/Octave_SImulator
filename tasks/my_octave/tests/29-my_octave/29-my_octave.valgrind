==12831== Memcheck, a memory error detector
==12831== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==12831== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==12831== Command: /home/irynuka1/Desktop/Programs/Tema2_1/check_my_octave_v1.1/check_my_octave/my_octave
==12831== Parent PID: 12830
==12831== 
==12831== 
==12831== HEAP SUMMARY:
==12831==     in use at exit: 0 bytes in 0 blocks
==12831==   total heap usage: 132 allocs, 132 frees, 10,036 bytes allocated
==12831== 
==12831== All heap blocks were freed -- no leaks are possible
==12831== 
==12831== For lists of detected and suppressed errors, rerun with: -s
==12831== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
