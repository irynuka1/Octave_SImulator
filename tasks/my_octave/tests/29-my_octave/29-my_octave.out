1167 2013 
1342 2326 
No matrix with the given index
1167 2013 
1342 2326 
30 18 26 
20 13 17 
20 10 18 
Power should be positive
30 18 26 
20 13 17 
20 10 18 
1780 1034 1554 
1200 699 1047 
1160 670 1014 
1780 1034 1554 
1200 699 1047 
1160 670 1014 
3 3
Cannot perform matrix multiplication
2 4 
1 3 
164 420 
105 269 
